packages/core/src/controllers/subscriber.ts

```diff
@@ -364,6 +389,7 @@ export class Subscriber extends ISubscriber {
       const batches = Math.ceil(this.cached.length / this.batchSubscribeTopicsLimit);
```

Comment:
Nit: `numOfBatches` would be clearer here to indicate that this is a number and not related the `batch` assignment below
```suggestion
      const numOfBatches = Math.ceil(this.cached.length / this.batchSubscribeTopicsLimit);
```